### PR TITLE
feat: render svg in clould function

### LINKED ISSUE #87: Sort through existing API logic and check for "computational" tasks (which can cause request blocking)
**TODO**
- [ ] GCP Cloud Function to generate SVG @baoyuantop

### PR DESCRIPTION
**Pull request by Baoyuantop**
resolve #87

### CODE REVIEW

**Review by LiteSun — APPROVED** [review]

File: api-go/tools/puppeteer/index.js [hunk 1]
```diff
@@ -7,14 +7,21 @@
...
 const { updateSeries } = require('./utils');
 const { JSDOM } = jsdom;
 
 const config = {
   width: 896,
   height: 550,
-}
+};
+
+const mergeRepoList = [
```
Comment (juzhiyuan) [inline api-go/tools/puppeteer/index.js:19]: Could we could maintain a json or yaml file to store this configuration?

Comment (Baoyuantop) [reply api-go/tools/puppeteer/index.js:19]: Both config and mergeRepoList ? @juzhiyuan 

I found [js-yaml](https://www.npmjs.com/package/js-yaml), after trying it feels good, we can use the YAML file.

**Review by juzhiyuan — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
Hi @Baoyuantop could you remove the title on the top-left corner, to keep the format with the original one? Or our users might face two different titles (one on the graph and one in markdown)
image

**Comment by Baoyuantop** [comment]
> Hi @Baoyuantop could you remove the title on the top-left corner, to keep the format with the original one? Or our users might face two different titles (one on the graph and one in markdown)
> image

let me check

**Pull request merged**